src/features/ci-cd/hooks/device-update-trigger.ts

```diff
@@ -10,10 +10,12 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 		// * app changed
 		// * target release changed
 		// * device name changed - so a user can restart their service and it will pick up the change
+		// * target OS release changed, so that Supervisors that support this can trigger a HUP
 		if (
 			(request.values.is_pinned_on__release !== undefined ||
 				request.values.belongs_to__application != null ||
-				request.values.device_name != null) &&
+				request.values.device_name != null ||
```

Comment:
I thought the supervisor only updated the device name for containers when they were receiving other updates? In which case there's no point to trigger a supervisor update check in that case as it would just be unnecessary load